vayesta/core/util.py

```diff
@@ -107,6 +107,34 @@ def func_with_doc(func):
 
 # --- NumPy
 
+def tril_indices_ndim(n, dims, include_diagonal=False):
```

Comment:
I would keep this function in the same file, until it is needed by multiple modules - unless you foresee that this will be the case

Reply:
It is a relatively long, general utility function, and if it is just defined in the types/wf/fci.py file, it would certainly be overlooked for other potential uses. If we want to do more >2nd order excitation methods, it would certainly be of use there. Unless you feel strongly, I would vote to leave it in a util.py file, which certainly feels like the right place.

Reply:
I think this is fine where it is.